examples/proto_dice/tests/proto_dice_tests.cpp

```diff
@@ -308,28 +305,29 @@ BOOST_FIXTURE_TEST_CASE(game_action_bad_state_test, proto_dice_tester) try {
 
     auto ses_id = new_game_session(game_name, player_name, casino_id, STRSYM("5.0000"));
 
-    game_action(game_name, ses_id, 0, { 30 });
+    game_action(game_name, ses_id, 0, {30});
 
-    BOOST_REQUIRE_EQUAL(push_action(game_name, N(gameaction), { player_name, N(game) }, { platform_name, N(active) }, mvo()
-        ("req_id", ses_id)
-        ("type", 0)
-        ("params", std::vector<param_t> { 30 })
-    ), wasm_assert_msg("state should be 'req_deposit' or 'req_action'"));
+    BOOST_REQUIRE_EQUAL(push_action(game_name,
+                                    N(gameaction),
+                                    {player_name, N(game)},
+                                    {platform_name, N(active)},
+                                    mvo()("req_id", ses_id)("type", 0)("params", std::vector<param_t>{30})),
```

Comment:
that's good, but really complicate to code reading

Reply:
Fixed manual